static/css/styles.css

```diff
@@ -1413,4 +1413,7 @@ header.masthead .masthead-heading {
   text-align: center;
   font-size: 0.9rem;
   font-family: "Montserrat", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+#navbarResponsive {
+  background-color: transparent;
```

Comment:
Why ?

Reply:
yesterday what I have suggested due to that some I am facing gray navbar but now I cloned and restart without this line now it working fine.
So, no need of this. I am removing this.